Gems/Atom/Asset/Shader/Code/AZSL/Platform/Android/Vulkan/AzslcHeader.azsli

```diff
@@ -18,4 +18,7 @@ static const float4 s_AzslDebugColor = float4(165.0 / 255.0, 30.0 / 255.0, 36.0
 
 // Uniform limitation need to be taken into consideration for mobile devices
 // [ATOM-14949]
-#define AZ_TRAIT_CONSTANT_BUFFER_LIMITATIONS 1
\ No newline at end of file
+#define AZ_TRAIT_CONSTANT_BUFFER_LIMITATIONS 1
```

Comment:
Should we have a comment here explaining what the limitation is? I assume bad things happen if not aligned to exactly a multiple of AZ_TRAIT_CONSTANT_BUFFER_ALIGNMENT?